core/src/sumeragi/fault.rs

```diff
@@ -744,7 +744,7 @@ impl<G: GenesisNetworkTrait, K: KuraTrait, W: WorldTrait, F: FaultInjection>
         }
 
         if let Err(error) = self.wsv.apply(block.clone()).await {
-            warn!(%error, %block_hash, "Failed to apply block on WSV");
+            warn!(?error, %block_hash, "Failed to apply block on WSV");
```

Comment:
why prefer debug vs display?

Reply:
If any information is missing from the display, we should probably add it to the `impl fmt::Display`, rather than use debugging in logs.

Reply:
While debugging I've found out, that debug print gives me more information. And this is `eyre::Report` so we can't implement `Display` on it. Probably it's time to make `WorldStateView` return `Result` with custom `Error` type